conf/messages.en

```diff
@@ -276,3 +276,31 @@ howManyCreditsForLostDamaged.error.lowBand.outOfMaxVal = Number of litres in the
 howManyCreditsForLostDamaged.error.highBand.outOfMaxVal = Number of litres in the high band must be less than 100,000,000,000,000
 howManyCreditsForLostDamaged.lowBand.change.hidden = lowBand
 howManyCreditsForLostDamaged.highBand.change.hidden = highBand
+
+howManyBroughtIntoTheUKFromSmallProducers.title = How many litres of liable drinks have you brought into the UK from small producers?
+howManyBroughtIntoTheUKFromSmallProducers.heading = How many litres of liable drinks have you brought into the UK from small producers?
+howManyBroughtIntoTheUKFromSmallProducers.subtext = Include your own brands of liable drinks produced outside of the UK.
+howManyBroughtIntoTheUKFromSmallProducers.lowBand = Litres in the low band
+howManyBroughtIntoTheUKFromSmallProducers.hint1 = At least 5 grams of sugar per 100 millilitres
+howManyBroughtIntoTheUKFromSmallProducers.highBand = highBand
+howManyBroughtIntoTheUKFromSmallProducers.hint2 = Litres in the high band
+howManyBroughtIntoTheUKFromSmallProducers.detailsLink = Liable drinks from small producers
+howManyAsAContract.detailsContent1 = If you are a registered small producer and you bring your own brand of liable drinks into the UK, you still need to report them but you will not pay the levy on them.
+howManyAsAContract.detailsContent2 = If you bring liable drinks into the UK from someone else who would be considered a small producer, you need to get evidence of the:
+howManyAsAContract.detailsContent3 = contact details, EU VAT number (if they have one) and website of the business
+howManyAsAContract.detailsContent4 = amount of litres of liable drinks packaged globally for brands the business owns in the past 12 months
+howManyAsAContract.detailsContent5 = signature of someone from the business, their position and the date of the signature
+howManyBroughtIntoTheUKFromSmallProducers.detailsContent = Details Contents
+howManyBroughtIntoTheUKFromSmallProducers.checkYourAnswersLabel = HowManyBroughtIntoTheUKFromSmallProducers
+howManyBroughtIntoTheUKFromSmallProducers.error.lowBand.required = Enter the number of litres in the low band
+howManyBroughtIntoTheUKFromSmallProducers.error.highBand.required = Enter the number of litres in the high band
+howManyBroughtIntoTheUKFromSmallProducers.error.lowBand.nonNumeric = Number of litres in the low band must be numeric
+howManyBroughtIntoTheUKFromSmallProducers.error.highBand.nonNumeric = Number of litres in the high band must be numeric
+howManyBroughtIntoTheUKFromSmallProducers.error.lowBand.negative = Number of litres in the low band must be 0 or more
```

Comment:
It's been decided to change this message to
Number of litres in the low band must be a positive number